ql/src/test/queries/clientpositive/external_part_parquet_int_decimal.q

```diff
@@ -0,0 +1,52 @@
+dfs ${system:test.dfs.mkdir} -p ${system:test.tmp.dir}/hive27213/ws_sold_date_sk=2451825;
```

Comment:
can you please clarify here why parquet_int_decimal.q doesn't fail without your patch?
with tables like:
```
CREATE TABLE parquet_decimal_spark (d1 decimal(4,2), d2 decimal(4,0), d3 decimal(12,5), d4 decimal(12,0)) stored as parquet;
```

Reply:
This particular case fails only with external partitioned table. Moreover it fails on select which is called after msck repair on partitioned table

Reply:
I see, but why is it behaving like this? msck makes sense, as without it the test doesn't even see the data I guess...also, regarding external table, it shouldn't be related to dictionaries, what's the connection between the table's managed/external nature and this issue? I guess it should reproduce the failure without partitions (copy the data file to a managed table's location with no partitions and define a managed table

Reply:
Yes! you are correct. This issue is neither related with external/manged table nor partitions/msck .
This is purely related to parquet file which has dictioanry defined. 

Answer to your question -why parquet_int_decimal.q doesn't fail without your patch?
>> Because parquet file used inside this q test does not have any encoding/dictionary defined inside it.

Attaching both parquet files and converted json files from which it shows attached dictionary in it.

I have changed q file names as now it is simple select on a table with dictionary supported parquet file